tests/test_websocket_list.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from websocket_list import WebSocketList
+
+
+@pytest.mark.asyncio
+async def test_get_ordered_by_ping_returns_single_server(backend_client):
```

Comment:
no longer ordered by ping

Reply:
What about a few items to test if the first one was returned instead of one-element list? If you want to test more cases pytest.mark.paramterize is nice way to go.

Reply:
Good catch. Renamed test name, also switched it to test for a list instead of a single entry.